frontend/src/components/common/StudentDetailCommon.vue

```diff
@@ -529,7 +540,10 @@ export default {
       traxStatus: '',
       loadingTraxData: false,
       loadingSchoolData: false,
-      compareStudent: []
+      compareStudent: [],
+      copyTxt: '',
```

Comment:
If variables like `copyTxt`, `dobCopy` and `diff` are just used in one method, they don't need to be defined in the `data()` method. It would be better to just define them in their own methods.